Assets/Shopify/UIToolkit/Theme Controllers/ThemeControllerBase.cs

```diff
@@ -48,5 +54,64 @@ public abstract class ThemeControllerBase : MonoBehaviour {
 
         public abstract void OnShow();
         public abstract void OnHide();
+        /// <summary>
+        /// The active cart that the controller is using.
+        /// </summary>
+        /// <returns>The Cart</returns>
+        public Cart Cart { get { return Client.Cart(); } }
```

Comment:
Nit: I would throw this stuff on a new line just to keep it in line with other getters we have in the SDK